docs/hazmat/primitives/asymmetric/dsa.rst

```diff
@@ -14,15 +20,18 @@ Generation
 
     .. versionadded:: 0.5
 
+    .. versionchanged:: 3.0
+
+        Added support for 4096-bit keys for some legacy applications that
+        continue to use DSA despite the wider cryptographic community's
```

Comment:
Technically our docs style guide says we shoudl cite things like that. Can you cite fillipo's latest post or something?